src/main/java/seedu/hustler/data/MemoryManager.java

```diff
@@ -0,0 +1,30 @@
+package seedu.hustler.data;
+
+import java.io.IOException;
+
+import seedu.hustler.Hustler;
+import seedu.hustler.ui.Ui;
+
+public class MemoryManager {
+
+    public MemoryManager() {
+
+    }
+
+    public void createBackup() {
+        Ui ui = new Ui();
+        Storage storage = new Storage();
+
+        try {
+            storage.createBackup(Hustler.list.return_list());
+            AvatarStorage.createBackup(Hustler.avatar);
+        } catch (IOException e) {
+
+        }
+    }
+
+    public void reloadBackup(int numberOfCommandsToUndo) {
+        Hustler.reloadBackup();
+        CommandLog.restoreData(numberOfCommandsToUndo);
+    }
```

Comment:
Missing header comments. All non trivial methods should have javadoc format header comments.